templates/server/src/main/resources/META-INF/archetype-post-generate.groovy

```diff
@@ -12,8 +12,8 @@ String batch = properties.get("batch")
 
 if (batch != "batch") {
   // delete folder batch recursively
-  Path rootPath = projectPath.resolve("batch")
-  deleteDirectoryRecursion(rootPath)
+  // Path rootPath = projectPath.resolve("batch")
+  // deleteDirectoryRecursion(rootPath)
 }
```

Comment:
Sure, I created this as draft PR because it is not complete. May suggestion is to remove batch support from archetype completely including this script. Maybe it will still be required for some other feature-toogle so I just did the min. change to make it work here.